### PR TITLE
Synergy-Induction_Programme Event Updated

### PR DESCRIPTION
**Pull request by TejasNayak42**
Make sure to change the Event thumbnail

### CODE REVIEW

File: src/contents/events/Synergy/cover.png
Comment (dev-shetty) [inline src/contents/events/Synergy/cover.png:1]: Change the cover image to synergy poster

File: src/contents/events/Synergy/index.md
Comment (dev-shetty) [inline src/contents/events/Synergy/index.md:1]: Try to add the images in grid in between the texts...looks too large rn

Comment (dev-shetty) [reply src/contents/events/Synergy/index.md:1]: Let it go in a sequence eg. SOSWC content with Nihara's photo explaining it

**Review by dev-shetty — CHANGES_REQUESTED** [review]

File: src/contents/events/Synergy/President.jpg
Comment (dev-shetty) [inline src/contents/events/Synergy/President.jpg:1]: Change my photo to other one, which is good please 😂😂😂

File: src/contents/events/Synergy/cover.jpg
Comment (dev-shetty) [inline src/contents/events/Synergy/cover.jpg:1]: Crop the poster a little, its getting cut in the final version

File: src/contents/events/Synergy/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Question and Answer Session:
+After the presentation, an interactive question and answer session was conducted. Participants had the opportunity to ask clarifying questions about SOSC, its activities, and the scope of involvement. The President and other team leads addressed the queries raised, ensuring a clear understanding of the organization's purpose and functioning.
+
+## Riddle Competition:
+To engage the participants and add an element of fun to the event, the team organized a riddle competition. The participants were challenged with a riddle, and the first person to solve it correctly would win a sticker as a prize. This activity encouraged active participation and created a lively atmosphere during the induction program.
+
+<!-- SOSWC Announcement:
+<Add things here> -->
+.
```
Comment (dev-shetty) [inline src/contents/events/Synergy/index.md]: Remove this period

File: src/contents/events/Synergy/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <p>
+      <img src="./SOSWCPresident.jpg" alt="SOSWC President" style="width: 100%;">
+    </p>
+    <p>
+      <img src="./Leads.jpg" alt="Tech Leads" style="width: 100%;">
+    </p>
+    <p>
+      <img src="./Interaction.jpg" alt="Interaction" style="width: 100%;">
+    </p>
+
```
Comment (dev-shetty) [inline src/contents/events/Synergy/index.md]: Just clean the alignment from here...

**Review by dev-shetty — CHANGES_REQUESTED** [review]

**Comment by TejasNayak42** [comment]
changes updated ;)

**Comment by TejasNayak42** [comment]
changes updated;)

**Review by dev-shetty — APPROVED** [review]

**Comment by dev-shetty** [comment]
LGTM ✅

Thank you for contributing :)

**Pull request merged**